app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -122,6 +128,30 @@ private void openDocument() {
         startActivityForResult(intent, ACTION_OPEN_DOCUMENT_REQUEST_CODE);
     }
 
+    private void setItem(MenuItem item, boolean enable) {
+        if (!enable) {
+            item.setEnabled(false);
+            item.getIcon().setAlpha(ALPHA_LOW);
+        } else {
+            item.setEnabled(true);
+            item.getIcon().setAlpha(ALPHA_HIGH);
+        }
+    }
+
+    private void checkButtonEnableDisable(MenuItem item, int zoomlevel, int desired_level) {
```

Comment:
This is going to need to be done when restoring the zoom level from savedInstanceState too. You can test by changing the orientation of the app by rotating the device, which will cause the activity to restart from saved state. This should probably just check the current zoom level and handle enabling / disabling both buttons. There will end up being other forms of zooming that aren't single steps up and down.